pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -266,7 +266,7 @@ func (rc *reconciler) mountAttachVolumes() {
 			}
 			if err == nil {
 				if remountingLogStr == "" {
-					klog.V(1).Infof(volumeToMount.GenerateMsgDetailed("operationExecutor.MountVolume started", remountingLogStr))
+					klog.Infof(volumeToMount.GenerateMsgDetailed("operationExecutor.MountVolume started", remountingLogStr))
```

Comment:
Nit: It feels kind of strange to use `Infof` when we know that `remountingLogStr == ""`